packages/dids/tests/methods/did-key.spec.ts

```diff
@@ -1,13 +1,14 @@
-import type { Jwk } from '@web5/crypto';
-
-import { expect } from 'chai';
-import { LocalKeyManager } from '@web5/crypto';
-
 import type { DidDocument } from '../../src/types/did-core.js';
+import type { Jwk } from '@web5/crypto';
 import type { PortableDid } from '../../src/types/portable-did.js';
 
+import chaiAsPromised from 'chai-as-promised';
```

Comment:
While communicated poorly perhaps the plan is to remove `chaiAsPromised` from all of the `@web5` packages to clear the way for being able to migrate off Mocha and Chai at some time.

```suggestion
```

Reply:
Hey @frankhinek, for my understanding:

1. What are we planning to migrate to?
2. What is the reason for the migration?

Reply:
@thehenrytsai
1. @shamilovtim had started working on evaluating options [here](https://github.com/TBD54566975/web5-js/issues/232#issuecomment-1826078772).
    - [TBD Testing Migration.pdf](https://github.com/TBD54566975/web5-js/files/13462108/TBD.Testing.Migration.pdf)
    - Its been on the backlog for a while and we need to make a decision.
2. Primary reasons for migration is test performance (mocha is slow compared to alternatives), limitations (e.g., [Watch mode](https://mochajs.org/#-watch-w) does not support ES Module test files), and general declining usage / retention (stateofjs.com).

It has not been a high priority since Mocha/Chai is functioning for our needs, but it would be great to have a timeline for when Web5 JS can address.